### PR TITLE
fix: Update article to configure a user namespace

### PR DESCRIPTION
**Pull request by tolusha**
## What does this pull request change?
fix: Update article to configure a user namespace

## What issues does this pull request fix or reference?
https://github.com/eclipse-che/che/issues/23176

## Specify the version of the product this pull request applies to
main

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [x] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Review by deerskindoll — APPROVED** [review]
lgtm

**Review by deerskindoll — APPROVED** [review]
+

File: modules/administration-guide/pages/configuring-a-user-namespace.adoc [hunk 1]
```diff
@@ -174,9 +174,122 @@ spec:
 ----
 ====
 
+. Create the `Template` below to replicate resources included in the Template to every user namespace.
```
Comment (deerskindoll) [inline modules/administration-guide/pages/configuring-a-user-namespace.adoc]:
```suggestion
. Create the `Template` below to replicate resources included in the Template to every user {namespace}.
```

File: modules/administration-guide/pages/configuring-a-user-namespace.adoc [hunk 2]
```diff
@@ -174,9 +174,122 @@ spec:
...
+- name: PROJECT_NAME
+- name: PROJECT_{prod-short}_USER
+
+----
++
+The `objects` array can contain any number of resources, including the previously mentioned ConfigMaps, Secrets, and PersistentVolumeClaims.
++
+The `parameters` are optional and define which parameters can be used. Currently, only `PROJECT_NAME` and `PROJECT_ADMIN_USER` are supported. `PROJECT_NAME` is the name of the {prod-short} {namespace}, while `PROJECT_ADMIN_USER` is the {prod-short} user of the {namespace}.
++
+The namespace name in objects is omitted and replaced with the user's namespace name during synchronization.
```
Comment (deerskindoll) [inline modules/administration-guide/pages/configuring-a-user-namespace.adoc]:
```suggestion
The {namespace} name in objects is omitted and replaced with the user's {namespace} name during synchronization.
```

**Review by deerskindoll — COMMENTED** [review]
Vale error fix. the pipeline should succeed with these changes.

File: modules/administration-guide/pages/configuring-a-user-namespace.adoc [hunk 3]
```diff
@@ -174,9 +174,122 @@ spec:
...
+  name: {prod-id-short}-user-namespace-configurator
+  namespace: {prod-namespace}
+  labels:
+    app.kubernetes.io/part-of: che.eclipse.org
+    app.kubernetes.io/component: workspaces-config
+objects:
+  ...
+parameters:
+- name: PROJECT_NAME
+- name: PROJECT_{prod-short}_USER
```
Comment (ibuziuk) [inline modules/administration-guide/pages/configuring-a-user-namespace.adoc]: @tolusha I do not quite understand why do we need extra parameters? Could you clarify the use-case for that? 
The original idea was just to do `oc process / oc apply` of the template for all user's namespaces

Comment (tolusha) [reply modules/administration-guide/pages/configuring-a-user-namespace.adoc]: `oc process` doesn't allow to keep updates up to date if Template has been changed.
We need manually create objects, watch for changes and update if needed

File: modules/administration-guide/pages/configuring-a-user-namespace.adoc [hunk 4]
```diff
@@ -174,9 +174,122 @@ spec:
...
+- name: PROJECT_NAME
+- name: PROJECT_ADMIN_USER
+
+----
++
+The `objects` array can contain any number of resources, including the previously mentioned ConfigMaps, Secrets, and PersistentVolumeClaims.
++
+The `parameters` are optional and define which parameters can be used. Currently, only `PROJECT_NAME` and `PROJECT_ADMIN_USER` are supported. `PROJECT_NAME` is the name of the {prod-short} {namespace}, while `PROJECT_ADMIN_USER` is the {prod-short} user of the {namespace}.
++
+The {namespace} name in objects is omitted and replaced with the user's {namespace} name during synchronization.
```
Comment (ibuziuk) [inline modules/administration-guide/pages/configuring-a-user-namespace.adoc]: if we want to promote those parameters in Eclipse Che docs we need to cross-link it with OpenShift docs.
I do not see any references in https://docs.openshift.com/container-platform/4.17/rest_api/template_apis/template-template-openshift-io-v1.html

**Comment by ibuziuk** [comment]
@deerskindoll do you happen to know why the preview URL is not working? https://github.com/eclipse-che/che-docs/pull/2807#issuecomment-2437230317

**Comment by deerskindoll** [comment]
> @deerskindoll do you happen to know why the preview URL is not working? [#2807 (comment)](https://github.com/eclipse-che/che-docs/pull/2807#issuecomment-2437230317)

@ibuziuk I suspect it's because of the failed pipelines. @mkuznyetsov  what do you think? can you please have a look? the asciidoctor issue was reported to the Vale support group but it didn't affect the other pipelines previously.

**Comment by deerskindoll** [comment]
update: running preview from a che workspace works. no idea why the preview isn't generated in the PR itself.

**Review by ibuziuk — APPROVED** [review]
👍

File: modules/administration-guide/pages/configuring-a-user-namespace.adoc [hunk 5]
```diff
@@ -174,9 +172,91 @@ spec:
...
+  metadata:
+    name: {prod-id-short}-user-rolebinding
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: Role
+    name: {prod-id-short}-user-roles
+  subjects:
+  - kind: User
+    apiGroup: rbac.authorization.k8s.io
+    name: ${PROJECT_ADMIN_USER}
```
Comment (themr0c) [inline modules/administration-guide/pages/configuring-a-user-namespace.adoc]:
```suggestion
    name: $\{PROJECT_ADMIN_USER}
```

**Pull request merged**